src/main/java/app/linguistai/bmvp/service/wordbank/UnknownWordService.java

```diff
@@ -273,7 +273,7 @@ public RUnknownWord addWord(QAddUnknownWord qAddUnknownWord, String email, boole
         } catch (WebClientResponseException e) {
             if (e.getStatusCode().equals(HttpStatus.NOT_FOUND)) {
                 log.error("Word {} is not found in the dictionary.", qAddUnknownWord.getWord());
-                throw new NotFoundException(String.format("Word %s is not valid.", qAddUnknownWord.getWord()));
+                throw new NotFoundException(String.format("It looks like %s isn't in our dictionary yet. Please double-check your spelling.", qAddUnknownWord.getWord()));
             }
             log.error("Dictionary service returned an error:" + e.getMessage());
             throw new SomethingWentWrongException();
```

Comment:
Would not this be better, as just returning SomethingWentWrong is so annoying

throw new SomethingWentWrongException("We had trouble communicating with our dictionary service, please try again later.")

Reply:
Very true! Changed it